epictrack-web/src/components/workPlan/status/StatusView/StatusHistory/index.tsx

```diff
@@ -116,24 +118,32 @@ const StatusHistory = () => {
         })}
         <When condition={approvedStatuses.length > SHOW_MORE_THRESHOLD}>
           <Collapse in={expand}>
-            {approvedStatuses.slice(SHOW_MORE_THRESHOLD).map((status) => (
-              <TimelineItem key={status.id}>
-                <TimelineOppositeContent>
-                  <ETPreviewText color={Palette.neutral.main}>
-                    <ReadMoreText>{status.description}</ReadMoreText>
-                  </ETPreviewText>
-                </TimelineOppositeContent>
-                <TimelineSeparator>
-                  <TimelineDot />
-                  <TimelineConnector />
-                </TimelineSeparator>
-                <TimelineContent>
-                  <ETCaption3 color={Palette.neutral.main}>
-                    {moment(status.posted_date).format(MONTH_DAY_YEAR)}
-                  </ETCaption3>
-                </TimelineContent>
-              </TimelineItem>
-            ))}
+            {approvedStatuses
+              .slice(SHOW_MORE_THRESHOLD)
+              .map((status, index) => {
+                const finalItem =
+                  approvedStatuses.length === index + 1 + SHOW_MORE_THRESHOLD;
+                return (
+                  <TimelineItem key={status.id}>
+                    <TimelineOppositeContent>
+                      <ETPreviewText color={Palette.neutral.main}>
+                        <ReadMoreText>{status.description}</ReadMoreText>
+                      </ETPreviewText>
+                    </TimelineOppositeContent>
+                    <TimelineSeparator>
+                      <TimelineDot />
+                      <When condition={!finalItem}>
```

Comment:
minor sugesstion, but try the Unless tag for this case instead of the when, it's cooler ; p